control-plane/subcommand/server-acl-init/command_test.go

```diff
@@ -1659,196 +1659,245 @@ func TestRun_ClientTokensRetry(t *testing.T) {
 // server tokens.
 func TestRun_AlreadyBootstrapped(t *testing.T) {
 	t.Parallel()
-	require := require.New(t)
-	k8s := fake.NewSimpleClientset()
-	setUpK8sServiceAccount(t, k8s, ns)
-
-	type APICall struct {
-		Method string
-		Path   string
+	cases := map[string]bool{
+		"token saved in k8s secret": true,
+		"token provided via file":   false,
 	}
```

Comment:
this test is mostly the same, except it now needs to test two cases instead of one (when token is stored as a secret and when it's provided via file)